File: myConnection.cs
using System.Data;
using MySql.Data.MySqlClient;

namespace MilkTeaOrderAndInventorySystem
{

    public class myConnection
    {
        private readonly MySqlConnection connection = new MySqlConnection("datasource=localhost;port=3306;username=root;password=0102bryan;database=accountdb");

        public MySqlConnection getConnection
        {
            get
            {
                return connection;
            }
        }

        public void openConnection()
        {
            if (connection.State == ConnectionState.Closed)
            {
                connection.Open();
            }
        }

        public void closeConnection()
        {
            if (connection.State == ConnectionState.Open)
            {
                connection.Close();
            }
        }

    }
}

File: myConnection.vb
Imports MySql.Data.MySqlClient

Public Class myConnection
    Private ReadOnly connection As New MySqlConnection("datasource=localhost;port=3306;username=root;password=0102bryan;database=accountdb")

    ReadOnly Property getConnection() As MySqlConnection
        Get
            Return connection
        End Get
    End Property

    Public Sub openConnection()
        If connection.State = ConnectionState.Closed Then
            connection.Open()
        End If
    End Sub

    Public Sub closeConnection()
        If connection.State = ConnectionState.Open Then
            connection.Close()
        End If
    End Sub

End Class
